### PR TITLE
Disable motion sensors and background sync by default

### PR DESCRIPTION
**Pull request by fmarier**
Fixes https://github.com/brave/brave-browser/issues/4709 and https://github.com/brave/brave-browser/issues/4789.

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:

For background sync:

1. Load https://deanhume.github.io/Service-Workers-BackgroundSync/
2. Open the devtools console.
3. Press the "Make an HTTP request - do it!" button.
4. Look for "Unable to fetch image." in the console (and not "sync registered").

For either feature:

1. Load https://brave.com
2. Click the padlock icon in the URL bar.
3. Click "Site settings".
4. Look for "Motion sensors" and "Background sync". They should both be labeled "Block (default)".

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [x] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.
  - [ ] Update https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove).

### CODE REVIEW

**Comment by bsclifton** [comment]
Adding @bridiver to review, since this involves patching

File: patches/components-content_settings-core-browser-content_settings_registry.cc.patch [hunk 1]
```diff
@@ -1,7 +1,25 @@
 diff --git a/components/content_settings/core/browser/content_settings_registry.cc b/components/content_settings/core/browser/content_settings_registry.cc
-index 03d0c89f73cb7a6a615195ca9ee36881fd4531c2..5e686c535d879f65f5a316041e9ad595d31a4285 100644
+index 03d0c89f73cb7a6a615195ca9ee36881fd4531c2..53f772a6d757d070b09ab2ddd5dba6dd24293d9b 100644
 --- a/components/content_settings/core/browser/content_settings_registry.cc
```
Comment (bridiver) [inline patches/components-content_settings-core-browser-content_settings_registry.cc.patch]: a patch already exists for this, please change the settings in `ContentSettingsRegistry::BraveInit` or better yet, clean up the existing patch by removing the patch on the header file and changing the patch in the cc file to be a macro

**Comment by fmarier** [comment]
@bridiver said that while we're in this patch, we should minimize this patch by getting rid of the [`.h` file](https://github.com/brave/brave-core/blob/master/patches/components-content_settings-core-browser-content_settings_registry.h.patch) by turning the [`BraveInit()` function](https://github.com/brave/brave-core/blob/master/patches/components-content_settings-core-browser-content_settings_registry.cc.patch) into a `BRAVE_INIT` macro.

**Comment by bsclifton** [comment]
cc: @fmarier per the feedback above (also needs a rebase)

File: patches/components-content_settings-core-browser-content_settings_registry.cc.patch (hunk 1 above)
Comment (fmarier) [reply patches/components-content_settings-core-browser-content_settings_registry.cc.patch]: I went for the macro and remove the patch for the `.h` file.

**Pull request merged**